infra/lambda_functions/rest/createrevision/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "on-connect",
+  "version": "1.0.0",
+  "description": "onconnect example for WebSockets on API Gateway",
+  "main": "src/app.js",
+  "author": "SAM CLI",
+  "license": "MIT",
```

Comment:
we haven't really discussed licenses... This is definitely something we should discuss